### PR TITLE
Label the kids throwing the ball back and forward a bit better. This …

### PR DESCRIPTION
**Pull request by daid**
…also labels two common used functions in bank 6, and removes some labels that are not references anywhere.

### CODE REVIEW

File: src/code/entities/bank6.asm [hunk 1]
```diff
@@ -89,7 +89,7 @@ SetEntitySpriteVariantForDirection::
     jp   SetEntitySpriteVariant                   ; $645A: $C3 $0C $3B
 
 ; Should Link talk to Entity
```
Comment (kemenaran) [inline src/code/entities/bank6.asm:91]:
```suggestion
; Should Link interact with Entity
```

File: src/code/entities/bank6.asm [hunk 2]
```diff
@@ -89,7 +89,7 @@ SetEntitySpriteVariantForDirection::
...
-func_006_645D::
+CheckLinkInteractionWithEntity_06::
```
Comment (kemenaran) [inline src/code/entities/bank6.asm:92]: I thought this was a variant of ShouldLinkTalkToEntity_**, but it actually does more thing, like checking for item usage ; so 👍

**Review by kemenaran — APPROVED** [review]
\o/

File: src/code/entities/bank6.asm [hunk 3]
```diff
@@ -33,7 +33,7 @@ include "code/entities/06_goponga_flower.asm"
 ;
 ; ----------------------------------------------------------------------
 
-func_006_641A::
+PushLinkOutOfEntity_06::
```
[the remark this answers is not in the document]
Comment (daid) [reply src/code/entities/bank6.asm:36]: I think my (new?) name is a bit better, as it's only solid for link, not anything else, and it's not really making it solid, just pushing link back.

**Review by kemenaran — APPROVED** [review]

**Pull request merged**